ruler-frontend/src/main/kotlin/com/spotify/ruler/frontend/components/Common.kt

```diff
@@ -43,37 +49,58 @@ import react.useState
 @RFunction
 fun RBuilder.report(report: AppReport) {
```

Comment:
If we support DFMs we also have to adapt the JSON report a little bit to include overall size information with and without DFMs. We could for example do this by adding `downloadSizeWithDynamicFeatureModules` and `installSizeWithDynamicFeatureModules` entries to the top-level app info.

In the HTML report we should probably have one global switch (checkbox) to switch between including or not including dynamic feature modules. Changing this switch should also change the overall values shown in the report and affect all pages, not only the breakdown (i.e. also the insights and ownership tabs).